rosa_release_notes/rosa-release-notes.adoc

```diff
@@ -16,6 +16,10 @@ toc::[]
 [id="rosa-q3-2023_{context}"]
 === Q3 2023
 
+* **ROSA CLI update:** The ROSA CLI (`rosa`) was updated to a new version. For information about what's changed in this release, see the link:https://github.com/openshift/rosa/releases/tag/v1.2.27[release notes]. For more information about the ROSA CLI (`rosa`), see xref:../cli_reference/rosa_cli/rosa-get-started-cli.adoc[Getting started with the ROSA CLI].
```

Comment:
We technically don't allow contractions so the "what's changed" reads a little odd, but this is also a "what's new" page so *shrug*